src/treq/response.py

```diff
@@ -91,3 +93,24 @@ def cookies(self):
                 jar.set_cookie(cookie)
 
         return jar
+
+    def check_status(self):
```

Comment:
Hmm... I think I agree as far as 'program to interfaces instead of classes' goes. But I'm not sure if `treq.content` is the right place, since the primary purpose is to check the status code and not the content.

But the only other place I could think to put it would be at the top level of `treq.request` which would make it the second top-level element.